mautic/mautic
# Contacts assigned to imported companies added to the selected segment.

| Q                                      | A
| -------------------------------------- | ---
| Branch?                                | "features" for 4.0
| Bug fix?                               | no
| New feature?                           | yes
| Deprecations?                          | no
| BC breaks?                             | no
| Automated tests included?              | yes
| Related user documentation PR URL      | 
| Related developer documentation PR URL | 
| Issue(s) addressed                     | 




#### Description:
We've added to company import segment mapping. These option allow contacts assigned to imported companies added to the selected segment. 



#### Steps to test this PR:
1. Load up [this PR](https://mautibox.com)
2. Create contacts and assign them to companies
3. Try import companies already created with assigned contacts  and select Contact segment during the import

![image](https://user-images.githubusercontent.com/462477/117329349-d8c34200-ae94-11eb-8900-4796954a9e6c.png)

4. Related contact should assigned to selected segment

![image](https://user-images.githubusercontent.com/462477/117329712-3788bb80-ae95-11eb-8684-4ff58fe9a742.png)






<a href="https://gitpod.io/#https://github.com/mautic/mautic/pull/9971"><img src="https://gitpod.io/button/open-in-gitpod.svg"/></a>
